Bottom: 1ac60e6db37c979817e8d5bf3172e758c15ffb19
Top:    1b6f46bd81a3d1093d292803e60903de15f4adf0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-27 20:05:19 +0000

Refresh of find-c-avoid-modifying-bounds-

---

diff --git a/src/find.c b/src/find.c
index e022312..7e93380 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2289,8 +2289,8 @@ LookupLOConnectionsToPolygon (PolygonType *Polygon, Cardinal LayerGroup)
   info.layer = LayerGroup;
   /* check rats */
   if (setjmp (info.env) == 0)
-    r_search (PCB->Data->rat_tree, &search_box,
-              NULL, LOCtoPolyRat_callback, &info);
+    r_search (PCB->Data->rat_tree, &search_box, NULL,
+              LOCtoPolyRat_callback, &info);
   else
     return true;
